### DISCUSSION TITLE
Switching from Community to Developer licence

### DISCUSSION THREAD

**Discussion by EngstromJimmy** [ravendb/ravendb#15843] [Q&A] [answered]
I am switching from Community to Developer license on our development instance (soon to be cluster). It seems like everything is still working. I am a bit worried about "Automatic renewal of Let’s Encrypt certificates is disabled" for the development license.

Is there anything I need to think about when it comes to switching from community to development licence?
What problems may occur?
The url will that stay the same?

Thanks
/Jimmy

**Comment by ayende** [comment, marked as answer]
The community and developer licenses use different domain names: `*.ravendb.community` vs. `*.development.run` 

You cannot renew a certificate on one using the other license.

Easiest way to handle this is to just run the setup process again, which will give you a URL that the server can renew.

**Reply by EngstromJimmy** [reply]
If the certificate isn't automatically updated, what do I need to make sure to do to have a valid certificate?

**Reply by garayx** [reply]
you can use the renew now button to manually renew the certificate: https://ravendb.net/docs/article-page/5.1/csharp/server/security/authentication/lets-encrypt-certificates#manual-renewal

**Reply by EngstromJimmy** [reply]
Thanks
One last question, "run the setup process again" how do I start that?
There are both databases and users on the server. Is there a way to keep them, or how do I solve that?
I have tried to find documentation on switching from community to Developer (or professional, which will be a later step in the process for us).

**Reply by ayende** [reply]
On your own machine, run the setup process again, and select "generate a setup package only" using the wizard and the dev license
that will generate a zip file that contains a `settings.json` and `pfx` file.
You can then update those files on the server, which will switch to the new URL.
